core-model/src/main/java/com/sequenceiq/cloudbreak/domain/stack/StackPatchType.java

```diff
@@ -0,0 +1,6 @@
+package com.sequenceiq.cloudbreak.domain.stack;
+
+public enum StackPatchType {
+    UNBOUND_RESTART,
+    UNKNOWN
```

Comment:
This is the fallback value for the enum converter - so that if we already save an enum value to the db but we have to roll back to a CB version that does not support that new enum value, it will not fail. See `StackPatchTypeConverter`.